src/lib.rs

```diff
@@ -6,17 +6,21 @@ extern crate serde_json;
 
 use chrono::{DateTime, Utc};
 use reqwest::header::{qitem, Accept, Authorization, Bearer, Link, RelationType, UserAgent};
+use reqwest::StatusCode;
 use serde_derive::Deserialize;
+use std::time::{UNIX_EPOCH, Duration, SystemTime};
 use std::{error, fmt, mem};
 
+
+
 #[derive(Debug)]
 pub struct Config {
     username: String,
     token: Option<String>,
 }
 
 impl Config {
-    fn url(self) -> Option<String> {
+    fn url(self) -> Option<String> { 
```

Comment:
```suggestion
    fn url(self) -> Option<String> {
```

nit: trailing whitespace